alvr/server_openvr/cpp/alvr_server/TrackedDevice.cpp

```diff
@@ -91,9 +83,40 @@ void TrackedDevice::submit_pose(vr::DriverPose_t pose) {
     );
 }
 
+bool TrackedDevice::register_device() {
+    if (!vr::VRServerDriverHost()->TrackedDeviceAdded(
+            this->get_serial_number().c_str(),
+            this->device_class,
+            (vr::ITrackedDeviceServerDriver*)this
+        )) {
+        Error("Failed to register device");
```

Comment:
This could maybe use some additional information like the serial number, to aid in debugging